pkgs/development/libraries/quarto/default.nix

```diff
@@ -0,0 +1,56 @@
+{ callPackage
+, lib
+, pandoc
+, esbuild
+, deno
+, stdenv
+, fetchurl
+, nixos
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "quarto";
+  version = "1.0.38";
+
+  src = fetchurl {
+    url = "https://github.com/quarto-dev/quarto-cli/releases/download/v1.0.38/quarto-1.0.38-linux-amd64.tar.gz";
+    sha256 = "sha256-bqEGRX6ihyrgmIxqkV9jr/2QfWM5LWnsjoGpTqK9qOc=";
+  };
+
+  buildInputs = [
+      pandoc
+      deno
+      esbuild
+  ];
+
+  installPhase = ''
+      mkdir src
+      tar -zxvf ${finalAttrs.src} -C src
```

Comment:
This should have already been performed by `unpackPhase`? Why did you do it manually?

Reply:
No proper reason, I missed the unpackPhase.

Reply:
Solved in [152051c1949b0717f59d072330ecc83bccd1fe6e](https://github.com/NixOS/nixpkgs/pull/186697/commits/152051c1949b0717f59d072330ecc83bccd1fe6e)